File: lab07/count_word.py
#!/usr/bin/env python3

import sys, re
count = 0
target = str.lower(sys.argv[1])

for line in sys.stdin:
   line = str.lower(line)
   words = re.split('[^a-zA-Z]+', line)
   #print(words)
   for word in words:
      if (word == target):
         count = count + 1

print("%s occurred %d times" % (target,count))

File: lab07/count_word.pl
#!/usr/bin/perl -w

$target = lc $ARGV[0];
$count = 0;

while ($line = <STDIN>) {
   @words = split(/[^a-z]+/i, lc $line);
   foreach $word (@words) {
      $count++ if ($word eq $target);
   }
}

print "$target occurred $count times\n ";
